capstone/src/main/java/capstone/Extras/Helper.java

```diff
@@ -236,6 +236,26 @@ public static String customer_search() {
     }
   }
 
+  public static Customer customer_search_new() {
+    while (true) {
+      // for admin and teller use
+      System.out.print("Enter the customer username: ");
+      String username = Helper.readLine();
+      Optional<Customer> queryRes = Database.getCustomer(username);
+      if (!queryRes.isEmpty()) {
+        return queryRes.get();
+      } else {
+        System.out.println(
+            ConsoleColours.RED_BOLD
+                + "This customer username does not exist! Please re-enter!"
+                + ConsoleColours.RESET
+                + "\uD83E\uDD7A");
+        Helper.pause();
+        continue;
```

Comment:
I think these two lines are not necessary. the `pause()` just adds an extra key press and the loop continues regardless of there being a `continue` keyword.

Reply:
fixed, thanks